index.md

```diff
@@ -1,4 +1,222 @@
 ---
 layout: docs
-title:  "Documentation Title"
----
\ No newline at end of file
+title:  "Guide to Digital Transfer"
+---
+## Requirements for the Transfer of Digital Records 
```

Comment:
This one is confusing because of the Docs Site structure, but the title in the front matter at the top of `index.md` is the h1 in the doc, so [I suggested](https://github.com/RockefellerArchiveCenter/digital-transfer-guide/pull/2#discussion_r1124885294) that this heading be deleted, and that there be one title in the front matter at that is either "Requirements for the Transfer of Digital Records" OR "Guide to Digital Transfer", but not both.